index.js

```diff
@@ -13,6 +13,10 @@ module.exports = {
         'index'
       ],
       pathGroups: [
+        {
+          pattern: '#+(imports|app)',
```

Comment:
Can we make this `#/**` ? I mean I think it would be okay to list all `#` prefixed stuff before `@` even if it's not coming from nuxt 👀

Reply:
Sure. Though `#/**` won't work for Nuxt as it targets path based imports and Nuxt's `#imports` resolves to a single module. Maybe we can have the combination so both are treated in the same respect?

Reply:
Let me commit to give you an idea.

Reply:
Ahhh you're right. Yeah in that case I think it would be fine like this. I haven't looked up but `#*` doesn't work right? 👀

Reply:
Yes, `#*` does work. We can stick to that for now? I don't think there is a use-case for `#/**` but we can adjust the pattern as and when needed?

Reply:
Nice! Yeah I think we can skip `#/*` for now. As you mentioned, we don't have a use case for at the moment 😳